hhast-lint.json

```diff
@@ -3,6 +3,8 @@
   "builtinLinters": "all",
   "extraLinters": ["Facebook\\HHAST\\Linters\\PreferLambdasLinter"],
   "disabledLinters": [
+    "Facebook\\HHAST\\Linters\\DontHaveTwoEmptyLinesInARowLinter",
+    "Facebook\\HHAST\\Linters\\NoNewlineAtStartOfControlFlowBlockLinter",
```

Comment:
the codegen ends up not passing these linters, but I don't think we care that much about these rules